src/handlers/assign.rs

```diff
@@ -47,16 +47,29 @@ impl Handler for AssignmentHandler {
         };
 
         if let Event::Issue(e) = event {
-            if e.action != github::IssuesAction::Opened {
+            if !matches!(e.action, github::IssuesAction::Opened | github::IssuesAction::Edited) {
                 log::debug!("skipping event, issue was {:?}", e.action);
-                // skip events other than opening the issue to avoid retriggering commands in the
+                // skip events other than opening or editing the issue to avoid retriggering commands in the
```

Comment:
Hm, so if we're correctly handling things, this should be entirely removable, right? Since we'll never retrigger unless the command has been edited.

Reply:
Well I though about doing so, but there are really a lot of event that targets issues. So I think its worth to skip the whole parsing when e.g. just labels changed